mozilla/multi-account-containers
# Adds a bookmark context menu for opening bookmarks in container tabs #323

Fix for #323 

# Steps to test
1. open bookmarks menu, sidebar, or library.
2. right click on a bookmark and select 'Open in container tab'

You can also right click a folder and open all the bookmarks in the same container tab from the folder.

One note: The context menu api does not allow a color to be set for the icon, so the only way I can see to colorize the icons in the context menu would be to make individual icons for each color, or I have seen an svg color filter, but we would still need one for each icon (and a filter for each color in use). I can do that if wanted.

# Screen capture
![Peek 2019-10-17 15-12](https://user-images.githubusercontent.com/3195251/67046134-45c98800-f0f5-11e9-9aaa-f41a2131d186.gif)
